Products/ZenCallHome/CallHomeStatus.py

```diff
@@ -76,45 +78,45 @@ def load_from_redis(self, k="CallHomeStatus"):
             log.warning(
                 "Exception trying to recive data from redis: {0}".format(e))
             self._redis_client = None
-            return
+            return None
```

Comment:
`return None`, while explicit, is redundant.  A bare `return` returns `None` by default.